spaceR/spaceR/Controllers/RocketsControlller.swift

```diff
@@ -5,55 +5,67 @@
 //  Created by Kieran Woodrow on 2022/02/25.
 //
 
-
 import Foundation
 import UIKit
 
 class RocketsController: UIViewController {
     
-    @IBOutlet weak var tableView: UITableView!
-    
-    @IBOutlet weak var emailIdLabel: UILabel!
-    var emailText = ""
-    private var rockets: [Rocket] = []
+    @IBOutlet weak var rocketTableView: UITableView!
+    private lazy var rocketViewModel = RocketViewModel()
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        self.emailIdLabel.text = self.emailText
-        getAllRockets()
-        setupTableView()
+        getAllRocketsFromAPI()
+        setTableView()
     }
     
-    private func setupTableView() {
-        self.tableView.delegate = self
-        self.tableView.dataSource = self
+    func setTableView() {
```

Comment:
Not sure why you removed the `private` for this function?

Reply:
Pretty much all the functions in this class can be private 😄

Reply:
okay cool